### DISCUSSION TITLE
Problem with authorization, error 403 forbidden

### DISCUSSION THREAD

**Discussion by StefanTegeltija** [community/community#40477] [Actions]
I have used this plugin: https://github.com/JoanEsquivel/cypress-course to make parallel execution of my cypress tests (I am using cypress 11.2) but I'm facing problem with authorization. On some of my tests Im visiting another URL(not URL) from my company and everything is working as expected for that one (so my settings are ok). Also please note that I can visit URL of my company from all browsers and devices, and from cypress UI (cypress open and cypress run are working perfectly). My company has some kind of firewall, which can recognise suspicion bot activities (obviously it is recognising GitHub action as "BOT"). Now I have to find the way to whitelist GitHub on some way, but I assume that GitHub changing/and using a lot of IP address... Also if I implement self hosted runner I will lose main purpose of this, what is to doing parallel execution of my tests on 5 machines (to make process faster) so I'm not sure is there is any workaround for this....do you know maybe the place in GitHub settings where I can change something to fix this?

**Comment by ossanna16** [comment]
Hi there @StefanTegeltija and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
